js/contact.js

```diff
@@ -0,0 +1,65 @@
+
+
+const submitContactForm = (form) => {
+    const username = form.username.value
+    const email = form.email.value
+    const message = form.message.value
+
+    console.log('username ', username, 'email ', email, ' message ', message)
+
+    const url = 'https://us-central1-youclap-dev.cloudfunctions.net/api/website/contact'
+
+    const data = {
+        'username': username,
+        'email': email,
+        'message': message
+    }
+
+    const formData = Object.keys(data)
+        .reduce((previous, current) => {
+            return previous.concat([current + '=' + data[current]])
+        }, [])
+        .join('&')
+
+    var htmlRequest = new XMLHttpRequest()
+
+    htmlRequest.onreadystatechange = (_event) => {
+        switch (htmlRequest.readyState) {
+            case htmlRequest.OPENED:
+                console.log('request opened')
+                break
+            case htmlRequest.LOADING:
+                console.log('request start loading...')
+                break
+            case htmlRequest.DONE:
+                form.reset()
```

Comment:
Add some notification for user receive feedback of success or in-success

Reply:
done 😛